test/unit/key_def.c

```diff
@@ -350,10 +458,11 @@ test_tuple_validate_key_parts_raw(void)
 static int
 test_main(void)
 {
-	plan(3);
+	plan(4);
 	header();
 
-	test_func_compare_with_key();
+	test_func_index_compare();
+	test_func_index_compare_with_key();
```

Comment:
Nit: the tests are about `key_def`s, not indexes, so `test_func_compare` or `test_func_tuple_compare` names would be better.

Reply:
Done.